src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -33,8 +33,20 @@
                    (request-handler request))
     (compojure/GET "/file_bucket_file/*" request
                    (request-handler request))
+
+    ;; TODO: file_bucket_file request PUTs from Puppet agents currently use a
```

Comment:
I added a commit, 7325909, which includes the ticket numbers for a couple of JIRA tickets I filed for the follow-on work - PUP-3812 for the core Puppet work and SERVER-294 to adapt Puppet Server to the core Puppet work if PUP-3812 is ever addressed.